deploy-manage/cloud-organization/billing/billing-faq.md

```diff
@@ -3,6 +3,9 @@ mapped_pages:
   - https://www.elastic.co/guide/en/cloud/current/ec-faq-billing.html
 applies_to:
   deployment:
+    self:
+    ece:
+    eck:
     ess: all
```

Comment:
This is probably just me not understanding how our applies_to tagging works. Should `ess` really be `all` when the others are blank?